src/Controller/ExportController.php

```diff
@@ -113,6 +114,11 @@ public function export(Request $request): Response
 
         $renderer = $this->export->getRendererById($type);
 
+        // display file inline if supported and `markAsExported` is not set
```

Comment:
I understand the choice in invoice screen where it makes sense, because the invoice listing is shown.
But what was your idea behind the check in exporter screen?

I guess it is "if you mark as export, you want to either share the document with your client or put it in your archive, and a direct download saves a couple of clicks", am I correct?

Reply:
Correct. That was my intention. Maybe I'm wrong here, but because there is no preview button, I thought about that if you don't check "mark as exported" you want to quick check you're timesheet. So in this case it is better to display the pdf inline.
But if this behavior is not logical enough for the user, I can remove this check. Otherwise there should be a separate button for this.

Reply:
I am not really using the export myself, so I can only make an educated guess. I'd say its good and if users do not get it, they will "post their complains" and we can work on improving it later on...